pages/positions/position.tsx

```diff
@@ -0,0 +1,55 @@
+import type {NextPage} from 'next'
+import {usePositionHandler} from '../../hooks/usePositionHandler'
+
+type PositionProps = {
+  positions: {asset: string; amountOfLiquidity: string; amountOfCollateral: string; profit: string; loss: string}[] | undefined
+}
+
+const PositionPage: NextPage = () => {
+  const {positions} = usePositionHandler()
```

Comment:
nit: make sure for any type of destructuring you have space in between, ex. `{ positions }`